pkg/api/handlers/compat/networks.go

```diff
@@ -22,6 +22,36 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
+type containerNetStatus struct {
+	name   string
+	id     string
+	status map[string]nettypes.StatusBlock
+}
+
+func getContainerNetStatuses(rt *libpod.Runtime) ([]containerNetStatus, error) {
```

Comment:
Nit: Remove get

Reply:
Are you fine with merging given all tests are green? I am good with changing but I rather avoid the force push unless necessary.